dygraphs/dygraphs.d.ts

```diff
@@ -0,0 +1,1272 @@
+// Type definitions for dygraphs
+// Project: http://dygraphs.com
+// Definitions by: Dan Vanderkam <http://danvk.org>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace dygraphs {
+  type DataArray = number[][];
+
+  // TODO: check for another DT definition to use here.
```

Comment:
I see that there are type definitions for gviz:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/c8e0d68ce3ee1758fdd468090d876eeeb94f2ccc/google.visualization/google.visualization.d.ts

If I add a `///` reference to this, will `typings` know about the dependency?